aw-server/src/android/mod.rs

```diff
@@ -93,33 +103,50 @@ pub mod android {
         string_to_jstring(&env, obj.to_string())
     }
 
+    lazy_static! {
+        static ref ASSET_PATH: RwLock<String> = RwLock::new(String::new());
```

Comment:
Why not simply use a Mutex here?

https://stackoverflow.com/questions/50704279/when-or-why-should-i-use-a-mutex-over-an-rwlock

Reply:
Didn't know much difference about those until you gave the ref; thanks ! Thought rwlock is rust's super safe method better than mutex, guess there are some tradeoffs :3

Reply:
Still didnt understand the difference 😕 but just impl'ed a simple mutex